src/block/utils/StructureBlockType.php

```diff
@@ -23,37 +23,11 @@
 
 namespace pocketmine\block\utils;
 
-enum StructureBlockType{
-	case DATA;
-	case SAVE;
-	case LOAD;
-	case CORNER;
-	case INVALID;
-	case EXPORT;
-
-	/**
-	 *  @throws \UnexpectedValueException
-	 */
-	public static function fromInt(int $type) : StructureBlockType{
-		return match($type){
-			0 => self::DATA,
-			1 => self::SAVE,
-			2 => self::LOAD,
-			3 => self::CORNER,
-			4 => self::INVALID,
-			5 => self::EXPORT,
-			default => throw new \UnexpectedValueException("Unknown structure block type " . $type),
-		};
-	}
-
-	public static function toInt(StructureBlockType $type) : int{
-		return match($type){
-			self::DATA => 0,
-			self::SAVE => 1,
-			self::LOAD => 2,
-			self::CORNER => 3,
-			self::INVALID => 4,
-			self::EXPORT => 5,
-		};
-	}
+enum StructureBlockType : int{
+	case DATA = 0;
+	case SAVE = 1;
+	case LOAD = 2;
+	case CORNER = 3;
+	case INVALID = 4;
+	case EXPORT = 5;
```

Comment:
The Bedrock values should not be exposed to the API.

Reply:
I've replaced it with normal enums with internal methods, what do you think ?